src/bridge/command.rs

```diff
@@ -102,23 +102,34 @@ fn platform_which(bin: &str) -> Option<String> {
     }
 }
 
-fn build_nvim_cmd_with_args(bin: &str) -> TokioCommand {
-    let mut args = vec!["--embed".to_string()];
-    args.extend(SETTINGS.get::<CmdLineSettings>().neovim_args);
-    let args_str = args.join(" ");
+#[cfg(target_os = "macos")]
+fn build_nvim_comd_with_args_platform(bin: &str, args: &[&str]) -> TokioCommand {
```

Comment:
The name for this function and its companion is slightly too long. Maybe `nvim_cmd_impl` would be better?